cmd/verify.go

```diff
@@ -159,20 +159,32 @@ func verify(cmd *cobra.Command, files []string) {
 		close(results)
 		recordReaderWg.Wait()
 
-		logger.WithFields(logrus.Fields{
+		if recordCount == 0 {
+			logrus.Errorf("No records present in files. Nothing has been checked.")
```

Comment:
as a matter of consistency, I think this should not be capitalized and avoid "."